### PR TITLE
Automatically clean activity log database

### LINKED ISSUE #4294: Jellyfin activity logs are retained indefinitely
### Context
The SQLite database `jellyfin.db` contains a table called `ActivityLogs` which records every user login/logout event, IP address, playback event, administrative action and so on and is used to display a list of the seven most recent activities on the admin dashboard and a list of the 100 most recent activities in the "Activity" menu option in the "Devices" category (see screenshots).

### Describe the bug
There is no retention policy for the rows in the `ActivityLogs` table, which leads to two problems:

1.  The size of the `jellyfin.db` file keeps growing indefinitely over time. On a Jellyfin instance that has been in use since November 2019, the `jellyfin.db` database has grown to a size of over 40MB and the `ActivityLogs` table accounts for almost 98% of that size at 39MB or about 236k rows.
2. In the event of a database breach or theft, the indefinite retention of user's IP addresses could have serious privacy implications, especially in Europe, where IP addresses are considered personal data under the GDPR.

### System
 - **OS:** Debian 10
 - **Virtualization:** Docker
 - **Clients:** Browser
 - **Browser:** Chromium 85
 - **Jellyfin Version:** 10.6.4

### Expected behavior
Since the Jellyfin web UI can only display the 100 most recent events from the `ActivityLogs` table, it would make sense for Jellyfin to automatically drop any entries which are older than that from the table instead of retaining them indefinitely. This could be done, for example, by adding a new scheduled task that runs every day.

### Screenshots
screenshot
screenshot2

### PR DESCRIPTION
**Pull request by crobibero**
Fixes #4294 
Note: the scheduled task does not have any triggers by default.

`ActivityLogRetentionDays` is nullable for API compatibility.

### CODE REVIEW

File: Emby.Server.Implementations/Localization/Core/en-US.json [hunk 1]
```diff
@@ -95,6 +95,8 @@
     "TasksLibraryCategory": "Library",
     "TasksApplicationCategory": "Application",
     "TasksChannelsCategory": "Internet Channels",
+    "TaskCleanActivityLog": "Clean Activity Log",
+    "TaskCleanActivityLogDescription": "Deletes activity log entries older then the configured age.",
```
Comment (cvium) [inline Emby.Server.Implementations/Localization/Core/en-US.json:99]:
```suggestion
    "TaskCleanActivityLogDescription": "Deletes activity log entries older than the configured age.",
```

**Review by cvium — APPROVED** [review]

**Review by EraYaN — APPROVED** [review]

**Review by Bond-009 — APPROVED** [review]
Documented code 😲

**Comment by ristein** [comment]
how can I change ActivityLogRetentionDays ?

**Comment by SircasticFox** [comment]
> how can I change ActivityLogRetentionDays ?

Anybody figured this out?

**Comment by alexkahler** [comment]
> > how can I change ActivityLogRetentionDays ?
> 
> Anybody figured this out?

Nope. I'm also trying to figure this out.

**Pull request merged**